server/webapp/WEB-INF/rails.new/app/views/layouts/agent_detail.html.erb

```diff
@@ -19,7 +19,6 @@
       <div class="page_header">
         <div class="row">
           <ul class="entity_title">
-            <!--<li class="name"><%= link_to l.string("AGENTS_POPUP_TITLE"), url_for(:controller => "agents", :action => "index") -%> </li>-->
```

Comment:
Link is already commented and we are not using it anymore to point to agents page. Thats why I removed it.